src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  box-sizing: content-box;
+  margin: 0;
+}
+
+.Search {
+  height: 70px;
+  width: 100vw;
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 4px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 2px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  position: relative;
+}
+
+.Search-Small {
+  height: 42px;
+}
+
+.imgSearch {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 23px;
+  z-index: 1;
+  border-radius: 0;
+}
+
+.imgSearch--Small {
+  position: absolute;
```

Comment:
You are setting position in imgSearch class which is common for both images, so there is no need to set it again here
```suggestion

```